gitbook/first-steps/erste-komponente.md

```diff
@@ -22,6 +22,9 @@ import 'antd/dist/antd.css';
 import './react-geo.css';
 ```
 
+If the `react-geo.css` file is not yet located in the `src` directory please
```

Comment:
Please add a short comment in https://github.com/terrestris/react-geo/issues/443 that this part will be modified if `css` file is included in dist folder of `react-geo`